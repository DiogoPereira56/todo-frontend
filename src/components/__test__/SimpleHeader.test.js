/* eslint-disable no-undef */
import SimpleHeader from '../SimpleHeader';
import ReactDOM from 'react-dom';
import { render } from '@testing-library/react';
//import 'jest-dom/extend-expect';
import renderer from 'react-test-renderer';

describe('<SimpleHeader />', () => {
    it('renders SimpleHeader without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<SimpleHeader />, div);
    });

    it('renders SimpleHeader correctly', () => {
        const header = render(<SimpleHeader />);
        expect(header);
    });

    it('matches SimpleHeader snapshot', () => {
        const tree = renderer.create(<SimpleHeader />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
