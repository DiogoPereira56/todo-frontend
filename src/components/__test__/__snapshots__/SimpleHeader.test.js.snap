// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleHeader /> matches SimpleHeader snapshot 1`] = `
<div
  className="sc-bdfBwQ kIYrtp"
>
  <div
    className="sc-gsTCUz ZQrBb"
  >
    <img
      alt=""
      className="sc-jSgupP kVoUAA"
      src="iconSquared.png"
    />
    <b>
      To Do
    </b>
  </div>
</div>
`;
